### PR TITLE
[s] I was bored

### PR DESCRIPTION
**Pull request by Incoming5643**
d10_icon

---

Found forevermore at the bottom right corner of z2

:heart:

### CODE REVIEW

**Comment by xxalpha** [comment]
where is the antag datum, you hack fraud

maxresdefault

**Comment by Incoming5643** [comment]
I was bored and **unproductive**.

Also going through some _life events_ that make finishing a large rewrite that will probably need to be babysat with patches for a few months a bit much. Just let me be dumb with small shit for now. The dream will never die, but my free time currently is.

**Comment by xxalpha** [comment]
Also I noticed you didn't use the smooth shuttle walls.

**Comment by Incoming5643** [comment]
They wouldn't smooth right with things like the "e"

**Comment by PKPenguin321** [comment]
:+1:

**Comment by lzimann** [comment]
the "H" bothers me a lot(in "the")

**Comment by Incoming5643** [comment]
The tragedy of using 3x4 letters, the other H works because they have half tile icon states.

**Comment by xxalpha** [comment]
Why not just use more space?

File: _maps/map_files/generic/z2.dmm [hunk 1]
```diff
@@ -1161,19 +1224,19 @@ abababababababababababababababababababababababababababababababababababababababab
...
-ababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababab
+abababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababrLmCrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrMrNab
```
Comment (KazeEspada) [inline _maps/map_files/generic/z2.dmm]: abababababababababababababa

Comment (xxalpha) [inline _maps/map_files/generic/z2.dmm]: Mr. Nab.

**Comment by Incoming5643** [comment]
Because then it wouldn't all fit on one vertical screen

**Comment by RemieRichards** [comment]
@Incoming5643 
isn't view 15x15?
You have a whole extra tile spare.

**Comment by Incoming5643** [comment]
But that would give "the coders" bigger text than "with love".

What are you implying

**Comment by ChangelingRain** [comment]
Add two half-tiles between the O, V, and E and move that entire top line like a tile to the right.

**Comment by Incoming5643** [comment]
the offset is intentional, the half tiles around the v could be fixed but maaaaaaaan what a pain.

**Comment by Incoming5643** [comment]
eh whatever I had my fun

**Pull request closed without merging**